src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -64,50 +62,92 @@ public class Intake extends SubsystemBase {
           .withSlot1(
               new Slot1Configs()
-                  .withKP(5)
-                  .withKD(0)
-                  .withKG(0)
-                  .withGravityType(GravityTypeValue.Arm_Cosine));
+                  .withKP(63.5)
+                  .withKD(12)
+                  .withKG(2)
+                  .withGravityType(GravityTypeValue.Arm_Cosine))
+          .withSlot2(
+              new Slot2Configs()
+                  .withKP(20.1)
+                  .withKS(1.8)
+                  .withKG(2)
+                  .withGravityType(GravityTypeValue.Arm_Cosine)
+                  .withStaticFeedforwardSign(StaticFeedforwardSignValue.UseClosedLoopSign));
 
   public static final CANcoderConfiguration intakeRotationSensorConfig =
       new CANcoderConfiguration()
           .withMagnetSensor(
               new MagnetSensorConfigs()
-                  .withMagnetOffset(0.001)
+                  .withMagnetOffset(0.085)
                   .withSensorDirection(SensorDirectionValue.Clockwise_Positive));
 
   public boolean up = false;
   public boolean stop = true;
 
+  private static final CANdiConfiguration intakeSensorsConfig =
+      new CANdiConfiguration()
+          .withDigitalInputs(
+              new DigitalInputsConfigs()
+                  .withS1CloseState(S1CloseStateValue.CloseWhenNotFloating)
+                  .withS1FloatState(S1FloatStateValue.FloatDetect)
+                  .withS2CloseState(S2CloseStateValue.CloseWhenNotFloating)
+                  .withS2FloatState(S2FloatStateValue.FloatDetect));
+
+  private static final TalonFXConfiguration intakeConfig =
+      new TalonFXConfiguration()
+          .withCurrentLimits(
+              new CurrentLimitsConfigs()
+                  .withStatorCurrentLimitEnable(true)
+                  .withStatorCurrentLimit(30))
+          .withSlot0(new Slot0Configs().withKS(5.4).withKP(3));
+
+  private static final NeutralOut motorStop = new NeutralOut();
+  private static final VelocityTorqueCurrentFOC intakeSpin =
+      new VelocityTorqueCurrentFOC(0).withAcceleration(IntakeConstants.intakeAcceleration);
+
+  private static final PositionTorqueCurrentFOC intakeMove = new PositionTorqueCurrentFOC(0);
+
   public Intake() {
     rotation.setNeutralMode(NeutralModeValue.Brake);
     intakeLeft.setNeutralMode(NeutralModeValue.Coast);
     intakeRight.setNeutralMode(NeutralModeValue.Coast);
 
+    intakeLeft.getConfigurator().apply(intakeConfig);
+    intakeRight.getConfigurator().apply(intakeConfig);
+
     rotation.getConfigurator().apply(intakeRotationConfig);
     rotationEncoder.getConfigurator().apply(intakeRotationSensorConfig);
 
     NamedCommands.registerCommand(
-        "IntakeDown", new IntakeMoveCommand(this, false, IntakeConstants.intakeDownAngle));
+        "IntakeDown", new IntakeMoveCommand(this, false, IntakeConstants.intakeDownAngle, 1, -3));
+    intakeSensors.getConfigurator().apply(intakeSensorsConfig);
```

Comment:
Nit: intakeSensors applying the configuration should probably be moved up above with the other config applying. I almost missed it because I didn't expect it to be after the registerCommand.